components/permit-holders/AppHistoryCard.tsx

```diff
@@ -0,0 +1,111 @@
+import { Box, Link, Text, Wrap, Divider } from '@chakra-ui/react'; // Chakra UI
+import Table from '@components/internal/Table'; // Table component
+import RequestStatusBadge from '@components/internal/RequestStatusBadge'; // Request status badge component
+import PermitHolderInfoCard from '@components/internal/PermitHolderInfoCard'; // Custom Card Component
+
+// Placeholder data
+
+const DATA = Array(4).fill({
+  rcdPermitId: { rcdPermitId: 354 },
+  isRenewal: 'false',
+  requestStatus: 'EXPIRING',
+  expiryDate: '2021/01/01',
+  applicationId: { applicationId: 1 },
+});
+
+const COLUMNS = [
+  {
+    Header: 'Permit #',
+    accessor: 'rcdPermitId',
+    disableSortBy: true,
+    maxWidth: 140,
+    Cell: _renderPermitId,
+  },
+  {
+    Header: 'Request Type',
+    accessor: 'isRenewal',
+    disableSortBy: true,
+    maxWidth: 140,
+    Cell: _renderRequestType,
+  },
+  {
+    Header: 'Status',
+    accessor: 'requestStatus',
+    disableSortBy: true,
+    maxWidth: 190,
+    Cell: _renderStatusBadge,
+  },
+  {
+    Header: 'Expiry Date',
+    accessor: 'expiryDate',
+    disableSortBy: true,
+    maxWidth: 140,
+  },
+  {
+    accessor: 'applicationId',
+    disableSortBy: true,
+    maxWidth: 140,
+    Cell: _renderAppLink,
+  },
+];
+
+type PermitIdProps = {
+  value: { rcdPermitId: number };
+};
+
+function _renderPermitId({ value }: PermitIdProps) {
+  return (
+    <>
+      <Text as="p">{`#${value.rcdPermitId}`}</Text>
+    </>
+  );
+}
+
+type RequestTypeProps = {
+  value: { isRenewal: boolean };
+};
+
+function _renderRequestType({ value }: RequestTypeProps) {
+  return (
+    <>
+      <Text as="p">{value.isRenewal ? `Renewal` : `Replacement`}</Text>
+    </>
+  );
+}
+
+type StatusProps = {
+  value: 'COMPLETED' | 'INPROGRESS' | 'PENDING' | 'REJECTED' | 'EXPIRING' | 'EXPIRED' | 'ACTIVE';
+};
+
+function _renderStatusBadge({ value }: StatusProps) {
+  return (
+    <Wrap>
+      <RequestStatusBadge variant={value} />
+    </Wrap>
+  );
+}
+
+type appProps = {
+  value: { applicationId: number };
+};
+
+function _renderAppLink({ value }: appProps) {
+  return (
+    <Link href={`/admin/request/${value.applicationId}`} passHref>
+      <Text textStyle="body-regular" textColor="primary" as="a">
+        View APP
+      </Text>
+    </Link>
+  );
+}
+
+export default function AppHistoryCard() {
+  return (
+    <PermitHolderInfoCard header={`APP History`} alignGridItems="normal">
+      <Divider pt="24px" />
+      <Box padding="20px 24px">
+        <Table columns={COLUMNS} data={DATA} />
```

Comment:
For this table as well as the ones in Attached Files and Medical History, there's no divider between the header and data. I personally kinda like the divider line and it would also be more simple to leave it since they use the reusable table component, but we should get a designer's input to see if this is okay

Reply:
Yep good idea! I'll message the designers

Reply:
Andy said it's we're all good to just keep the divider for now